tests/test_db.py

```
@@ -0,0 +1,43 @@
+# test_db.py
+
+import unittest
+from peewee import *
+
+from app.models.timelinepost import TimelinePost
+
+
+MODELS = [TimelinePost]
+
+test_db = SqliteDatabase(':memory:')
```

Ruff (0.12.2)

`SqliteDatabase` may be undefined, or defined from star imports

(F405)